cql3/statements/create_table_statement.cc

```diff
@@ -185,8 +185,17 @@ std::unique_ptr<prepared_statement> create_table_statement::raw_statement::prepa
     if (_properties.properties()->get_synchronous_updates_flag()) {
         throw exceptions::invalid_request_exception(format("The synchronous_updates option is only applicable to materialized views, not to base tables"));
     }
+    std::vector<sstring> stmt_warnings;
+    auto stmt_warning = [&] (sstring msg) {
+        if (this_shard_id() == 0) {
+            mylogger.warn("{}: {}", cf_name, msg);
+        }
```

Comment:
Why is this in prepare and not in execute?

Reply:
execute is implemented by the super class: schema_altering_statement::execute.
So ai followed the existing design of where the checks are implemented.

Reply:
By the way, if we can find an error during prepare (because it's not related to prepared-statement parameters, etc.), isn't it better? Why allow the prepare but then fail the execute? Clearly, for "syntax error", the prepare itself fails, so it's probably good to fail prepare on some of the other types of errors as well.